include/seqan3/alignment/pairwise/edit_distance_unbanded.hpp

```diff
@@ -710,6 +718,18 @@ class edit_distance_unbanded :
     using edit_traits::word_size;
 
 private:
+
+    template <typename, typename>
+    friend class edit_distance_unbanded_max_errors_policy;
```

Comment:
same here. Actually didn't doxygen trigger an warning if the friend declaration is not documented?

Reply:
A block comment would be ok here I guess